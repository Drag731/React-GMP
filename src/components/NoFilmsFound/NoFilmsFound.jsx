import React from 'react';

import './NoFilmsFound.scss'

const NoFilmsFound = () => (
    <div className="b-not-found">
        <div className="b-not-found__wrap">
            <span className="b-not-found__text">No films found</span>
        </div>
    </div>
);

export default NoFilmsFound;
