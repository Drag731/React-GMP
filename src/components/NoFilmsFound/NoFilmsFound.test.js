import React from 'react';
import renderer from 'react-test-renderer';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import NoFilmsFound from './NoFilmsFound'

configure({ adapter: new Adapter() });

describe('<NoFilmsFound  />', () => {
    it('Snapshot one', () => {
        const tree = renderer
            .create(<NoFilmsFound />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('Renders NoFilmsFound component', () => {
        const tree = shallow(
            <NoFilmsFound />
        );
        expect(tree.find('.b-not-found')).toHaveLength(1);
    });
});
