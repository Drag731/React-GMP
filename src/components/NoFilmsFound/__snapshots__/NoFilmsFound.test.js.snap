// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoFilmsFound  /> Snapshot one 1`] = `
<div
  className="b-not-found"
>
  <div
    className="b-not-found__wrap"
  >
    <span
      className="b-not-found__text"
    >
      No films found
    </span>
  </div>
</div>
`;
